ADAL/src/broker/ios/ADBrokerKeyHelper.m

```diff
@@ -328,7 +328,7 @@ + (NSDictionary *)decryptBrokerResponse:(NSDictionary *)response correlationId:(
     }
 
     //now compute the hash on the unencrypted data
-    NSString *actualHash = [ADPkeyAuthHelper computeThumbprint:decrypted isSha2:YES];
+    NSString *actualHash = [MSIDPkeyAuthHelper computeThumbprint:decrypted isSha2:YES];
```

Comment:
That's funny that we have Sha2 computation only inside PkeyAuthHelper :)

Reply:
:neckbeard: